### PR TITLE
Broken link questions in devguide

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/hardware/porting_guide.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To run `qconfig` you may need to install additional Qt dependencies.
 Linux boards do not include the OS and kernel configuration. 
 These are already provided by the Linux image available for the board (which needs to support the inertial sensors out of the box).
 
-* [cmake/configs/posix\_rpi\_cross.cmake](https://github.com/PX4/Firmware/blob/master/cmake/configs/posix_rpi_cross.cmake) - RPI cross-compilation.
+* [cmake/configs/posix\_rpi\_cross.cmake](https://github.com/PX4/Firmware/blob/master/cmake/configs/posix_rpi_cross.cmake) - RPI cross-compilation.
```
Comment (hamishwillee) [inline en/hardware/porting_guide.md]: @dagar Where did these move?

File: en/hardware/porting_guide.md [hunk 2]
```diff
@@ -90,17 +90,17 @@ This section describes the various middleware components, and the configuration
 
 * The start script is located in [posix-configs/](https://github.com/PX4/Firmware/tree/master/posix-configs).
 * The OS configuration is part of the default Linux image (TODO: Provide location of LINUX IMAGE and flash instructions).
-* The PX4 middleware configuration is located in [src/drivers/boards](https://github.com/PX4/Firmware/tree/master/src/drivers/boards). TODO: ADD BUS CONFIG
+* The PX4 middleware configuration is located in [src/drivers/boards](https://github.com/PX4/Firmware/tree/master/src/drivers/boards). TODO: ADD BUS CONFIG
```
Comment (hamishwillee) [inline en/hardware/porting_guide.md]: @dagar Where did https://github.com/PX4/Firmware/tree/master/src/drivers/boards move?

File: en/hardware/porting_guide.md [hunk 3]
```diff
@@ -90,17 +90,17 @@ This section describes the various middleware components, and the configuration
...
 
 ## RC UART Wiring Recommendations
 
-It is generally recommended to connect RC via separate RX and TX pins to the microcontroller. 
+It is generally recommended to connect RC via separate RX and TX pins to the microcontroller.
 If however RX and TX are connected together, the UART has to be put into singlewire mode to prevent any contention. 
-This is done via board config and manifest files. 
-One example is [px4fmu-v5](https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v5/manifest.c).
+This is done via board config and manifest files.
+One example is [px4fmu-v5](https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v5/manifest.c).
```
Comment (hamishwillee) [inline en/hardware/porting_guide.md]: @dagar Where did https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v5/manifest.c move?

File: en/hardware/porting_guide.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ For example, for FMUv5:
   - A board-specific initialisation file is automatically included in startup scripts if found under the boards directory at **init/rc.board**.
   - The file is used to start sensors (and other things) that only exist on a particular board. 
     It may also be used to set a board's default parameters, UART mappings, and any other special cases.
-  - For FMUv5 you can see all the Pixhawk 4 sensors being started, and it also sets a larger LOGGER_BUF, and in AUTOCNF section (fresh setups) it sets the [SYS_FMU_TASK](../advanced/parameter_reference.md#SYS_FMU_TASK) parameter.
+  - For FMUv5 you can see all the Pixhawk 4 sensors being started, and it also sets a larger LOGGER_BUF, and in AUTOCNF section (fresh setups) it sets the [SYS_FMU_TASK](../advanced/parameter_reference.md#SYS_FMU_TASK) parameter.
```
Comment (hamishwillee) [inline en/hardware/porting_guide.md]: This does not exist in master (SYS_FMU_TASK) - was it replaced with anything or deleted (ie do I just delete this line?)

**Comment by hamishwillee** [comment]
@bkueng If you have advice on where these build file might have gone would be appreciated.

**Comment by bkueng** [comment]
The configs are under /boards: https://github.com/PX4/Firmware/blob/master/boards/px4/raspberrypi/cross.cmake
And board-specific code: https://github.com/PX4/Firmware/tree/master/boards/px4/raspberrypi/src

**Comment by hamishwillee** [comment]
Thanks @bkueng , much appreciated.

**Pull request merged**